src/test/resources/org/javarosa/xform/parse/internal_empty_select.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:odk="http://www.opendatakit.org/xforms" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <h:head>
+        <h:title>internal empty select</h:title>
+        <model>
+            <instance>
+                <internal_empty_select id="internal_empty_select">
+                    <first/>
+                    <meta>
+                        <instanceID/>
+                    </meta>
+                </internal_empty_select>
+            </instance>
+            <instance id="first">
+                <root>
+                </root>
+            </instance>
+            <bind nodeset="/internal_empty_select/first" type="select1"/>
+            <bind calculate="concat('uuid:', uuid())" nodeset="/internal_empty_select/meta/instanceID" readonly="true()" type="string"/>
+        </model>
+    </h:head>
+    <h:body>
+        <select1 ref="/internal_select_10/first">
```

Comment:
The root node name here should be `internal_empty_select`!

Reply:
Should now be resolved